docs/changelog/2766.feature.rst

```diff
@@ -0,0 +1,2 @@
+* Allow ``-f`` to be used multiple times and to be used on hyphenated factors
```

Comment:
Remove the leading start, and finish your sentence with a period.

Reply:
Also add by user directive at the end, see other entries in the changelog template.

Reply:
I wasn't sure what you meant, but judging from the template it looks like `- ` gets prepended, so I remove the `*` bullet-point.

I also did a little rephrase to be shorter and fixed the missing annotation.